user-guide/Standard_Apps/SRM/SRM_migrating/SRM_migrating_booking_manager_full_configuration.md

```diff
@@ -0,0 +1,121 @@
+---
+
+# Migrating a complete Booking Manager configuration
+
+The SRM framework allows you to export all artifacts of a Booking Manager application to facilitate deployment. You can then add the configuration to a Companion Files repository so it can be included in an install package, as detailed below.
+
+This functionality is available since version 1.2.24 of the SRM framework.
+
+Consider deploying the *Ziine SatUplink* booking application.
+
+## Exporting the configuration
+
+To export the configuration related to a specific Booking Manager application and its virtual platform, use the *SRM_ExportFullConfiguration* Automation script, which is included in the SRM package.
+
+1. In the Automation module, select the *SRM_ExportFullConfiguration* script and click *Execute*.
+
+1. As the input data for the script, specify a JSON object that defines which Booking Manager application should be exported.
+
+   For example:
+
+   ```json
+   { "BookingManagerName" : "Ziine SatUplink" }
+   ```
+
+1. Click *Execute now* to start the script.
+
+When the configuration has been exported, it will be available as a zip file in the *Skyline Booking Manager* > *Full Configurations* documents folder. The file name consists of the specified Booking Manager and the export time, e.g. "Ziine SatUplink_\`export time\`".
+
+The export will include:
+
+- The Booking Manager element, including:
+
+  - Its settings
+
+  - The custom Visio drawing
+
+  - The custom Automation scripts defined for the actions and related to the custom actions
+
+  - The view containing the Booking Manager element
+
+  - The views and the Visio drawings used in the custom actions
+
+  - The alarm and trend template
+
+- The related booking monitoring element, view, alarm template, and trend template
+
+- Service definitions, based on the virtual platform they are linked to
+
+  - The service Visio drawing set in the definition
+
+  - The functions used in the service definitions, including their profile parameters, profile definitions, and profile load scripts
+
+  - The resource pools referenced in the service definition, including inheritance, *Resource pool* property, *Contributing Configuration* property, and *Path* property
+
+  - Resources used for inheritance and the parameters they reference
+
+  - Profile instances linked to the profile definitions of the functions referenced in the service definitions
+
+- Resource pools related to the virtual platform
+
+## Including the configuration in a .dmapp package
+
+### Including the configuration in a 'Companion Files' repository
+
+To include the exported configuration in a package, the files in the exported zip file should be added to the *Companion Files* repository. After adding that repository and cloning it, create a folder called "SRM" in the *Content* folder of that repository. Next, export the folders inside the exported zip file to that newly created *SRM* folder. Commit these files and push the repository.
```

Comment:
This is a bit too vague: where can people find this repository? Is this for Skyline users only, or also for outside users?